department-of-veterans-affairs/caseflow
# Worksheet Print pdf cleanup

Resolves #4121 

### Description
This removes all the extra margins on in table and extra space that was on the second page. I also managed to create extra appeals for testing purposes.
### Acceptance Criteria 
- [ ] Reduce extra spacing between rows.
- [ ] Reduce the extra spacing on the second page
- [ ] Added more Appeal Streams.
### Testing Plan
1. Go to Worksheet print page
2. Inspect the table to see if the spaces were removed
3. Check second page to see if extra space was removed on the form.

<img width="908" alt="screen shot 2018-03-29 at 10 09 41 am" src="https://user-images.githubusercontent.com/23080951/38093633-8df7a97a-3339-11e8-8876-2614f3afc475.png">
